int32 index
sensor_msgs/Image descriptor
